cleanlab_studio/studio/trustworthy_language_model.py

```diff
@@ -276,3 +280,13 @@ async def get_confidence_score_async(
                     )
                 )["confidence_score"],
             )
+
+
+def is_notebook() -> bool:
+    """Returns True if running in a notebook, False otherwise."""
+    try:
+        get_ipython = sys.modules["IPython"].get_ipython
+        if "IPKernelApp" in get_ipython().config:
+            return True
+    finally:
+        return False
```

Comment:
Seems like this always returns False because of the `finally` block

Reply:
🤲 

good catch. shouldn't have tried to clean up the method w/o testing again 😢